### ISSUE TITLE
is lumagl.min.js available?

### ISSUE THREAD

**Issue by SymbolixAU** [uber/luma.gl#927]
#### Background 

There are a couple of references to minified luma.gl, for example #734,  #704 and #234. Is the .min.js available somewhere (similar to deck.gl "https://unpkg.com/deck.gl@latest/deckgl.min.js")? 

#### To Do List 
  - [x] Add label and assign to milestone 
  - [ ] Coding
  - [ ] Doc update
  - [ ] Whats new update 
  - [ ] Test

**Comment by jianhuang01** [comment]
@SymbolixAU The tickets you referenced are not talking about lumagl.min.js, which are focusing on reducing the size of luma.gl npm library. We didn't publish the lumagl.min.js yet but there is a submodule [@luma.gl/scripts](https://github.com/uber/luma.gl/tree/master/modules/script) for that purpose. You can generate lumagl.min.js by building our luma.gl project.

**Comment by SymbolixAU** [comment]
Thanks - yes the lumagl.min.js is generated when I build the library locally. 

(I get another issue about conflicting versions, but I think I can solve that)

**Comment by SymbolixAU** [comment]
actually I'm getting a different error 

![screen shot 2019-03-05 at 8 14 02 am](https://user-images.githubusercontent.com/18344164/53763383-baf99b80-3f1e-11e9-8990-8f29de7c30b5.png)

I'm using deck.gl v6.4.2 (https://unpkg.com/deck.gl@6.4.2/deckgl.min.js) and luma.gl release [v6.4.2 - 9ddd463](https://github.com/SymbolixAU/luma.gl/releases)

**Comment by ibgreen** [comment]
@SymbolixAU Somehow webpack is loading an installed version of luma.gl in parallel with your script. You probably have code that imports luma.gl somewhere in your code.

Normally luma.gl is included in the `deck.min.js` script. Are you trying to use luma.gl standalone?

**Comment by SymbolixAU** [comment]
@ibgreen What I'm actually trying to do is build the [trips layer](https://github.com/uber/deck.gl/tree/master/modules/experimental-layers/src/trips-layer) using pure js. However, I got an error saying `Model` is not available (from luma.gl), so I was seeing if I could use the luma.js directly. 

However, I see you're doing a big restructure of deck.gl anyway for v7.0 (and the experimental/trips layer) has moved (been removed?). So, I might way for v7 to be released before trying again.

**Comment by ibgreen** [comment]
If you are using pure js using the deck.gl script, then you may find `Model` on the global `luma` variable.

https://github.com/uber/deck.gl/blob/7890d05e3343c78a8a4e20eb6b2019addb3a02cf/modules/lite/src/index.js#L17

If you are using pure js (not react) using imports, just import it from luma.gl

**Comment by SymbolixAU** [comment]
Thanks @ibgreen, you are correct, I can use `luma.Model` (and therefore `luma.Geometry`) directly, without needing the lumagl.min.js loaded separately.

Therefore @jianhuang01 I think my original issue is closed as I can generate the min.js myself.
